content/api/commands/as.md

```diff
@@ -58,14 +58,21 @@ it('disables on click', () => {
 
 ### Route
```

Comment:
Should we change the header from `Route` to `Intercept`?

Reply:
It depends on their definitions which are currently unclear.